schema/vrs-source.yaml

```diff
@@ -240,6 +246,43 @@ definitions:
 
     required: [ "subject", "copies" ]
 
+  Genotype:
+    description: >-
+      A set of trans-phased :ref:`MolecularVariation` members, with associated
+      copy counts, across a specified number of genomic locus `copies`.
+    additionalProperties: false
+    type: object
+    properties:
+      type:
+        type: string
+        const: "Genotype"
+        default: "Genotype"
+        description: >-
+          MUST be "Genotype"
+      members:
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          $ref: "#/definitions/GenotypeMember"
+        description: >-
+          Each GenotypeMember in `members` describes a :ref:`MolecularVariation`
```

Comment:
This is a slightly better definition than the one that is currently associated with the GenotypeMember section.